QuizUp/actions/upload_to_s3.rb

```diff
@@ -0,0 +1,32 @@
+module Fastlane
+  module Actions
+    class UploadToS3Action < Action
+      def self.run(params)
+
+        s3 = Aws::S3::Resource.new
+        file_path = File.expand_path(File.dirname(__FILE__) + "/../../QuizUp.ipa")
```

Comment:
This should be an option, that has the default value of `Actions.lane_context[SharedValues::IPA_OUTPUT_PATH]`